201-front-door-standard-premium-rule-set/prereqs/prereq.azuredeploy.bicep

```diff
@@ -0,0 +1,106 @@
+@description('The location into which the Azure Storage resources should be deployed.')
+param location string = resourceGroup().location
+
+@description('The name of the Azure Storage account to create. This must be globally unique.')
+param accountName string = 'stor${uniqueString(resourceGroup().id)}'
+
+@description('The name of the SKU to use when creating the Azure Storage account.')
+@allowed([
+  'Standard_LRS'
+  'Standard_GRS'
+  'Standard_ZRS'
+  'Standard_GZRS'
+  'Premium_LRS'
+])
+param skuName string = 'Standard_LRS'
+
+@description('The name of the page to display when a user navigates to the root of your static website.')
+param indexDocument string = 'index.htm'
+
+@description('The name of the page to display when a user attempts to navigate to a page that does not exist in your static website.')
+param errorDocument404Path string = '404.htm'
+
+var storageAccountContributorRoleDefinitionId = subscriptionResourceId('Microsoft.Authorization/roleDefinitions', '17d1049b-9a84-46fb-8f53-869881c3d3ab') // as per https://docs.microsoft.com/en-us/azure/role-based-access-control/built-in-roles#:~:text=17d1049b-9a84-46fb-8f53-869881c3d3ab
+var storageAccountStorageBlobDataContributorRoleDefinitionId = subscriptionResourceId('Microsoft.Authorization/roleDefinitions', 'ba92f5b4-2d11-453d-a403-e96b0029c9fe') // as per https://docs.microsoft.com/en-us/azure/role-based-access-control/built-in-roles#:~:text=ba92f5b4-2d11-453d-a403-e96b0029c9fe
+var managedIdentityName = 'StorageStaticWebsiteEnabler'
+var deploymentScriptName = 'EnableStorageStaticWebsite'
+
+resource storageAccount 'Microsoft.Storage/storageAccounts@2019-06-01' = {
+  name: accountName
+  location: location
+  kind: 'StorageV2'
+  sku: {
+    name: skuName
+  }
+  properties: {
+    supportsHttpsTrafficOnly: false // This is only configured to make this sample work correctly with Application Gateway. This is not recommended practice - for production solutions you should always use end-to-end HTTPS.

```

Comment:
why can't we set it to true?  and show users how to do that?

Reply:
Sorry - that was a holdover from an earlier version of the module. I've removed it.